### ISSUE TITLE
Associate pre-existing issues with appropriate repo(s)

### ISSUE THREAD

**Issue by ellerychan** [brata-hsdc/brata.masterserver#3]
The issues ported from Google Code have tags saying which sub-project they go with (like `Component-MS`.  Someone should figure out how to associate them with the `brata.masterserver` repo, so they show up in the list attached to the repo, like _this_ issue does.

**Comment by DragonmasterJ** [comment]
I think we can link issue numbers across projects by using the hashtag, and using the reponame/issue number. Within a project, to link an issue I think you use the hashtag followed by the issue number. 

Any new issues in the brata I think we can make new issues in this project and in the description just link the old issue. 
For the older ones, maybe we can do the same thing?

**Comment by ellerychan** [comment]
Sounds good, but I'm not sure I completely understand. I'm going to do a trial run with our favorite issue, I guess like this: brata-hsdc/brata#74?  Are you saying you will just open it as a new issue here and just put this reference in it back to the old one.  Then that one becomes dormant.   We should also put a forward pointer from the old to the new issue.

Do you concur with that approach?

**Comment by jawaad-ahmad** [comment]
Agreed. Proposed process:
- Find desired issue in brata repo.
- Open corresponding issue in brata.masterserver repo.
  - Same headline.
  - Link to original issue in the comments.
  - Same tags.
  - Note the new issue number, URL, etc.
- Update the original issue in the brata repo.
  - Link to new issue in the comments.
  - Advance its state to resolved.

I’d like to keep the labels consistent across all repositories, so please let me know if we’re indeed doing this and I can take the action to replicate the issue labels.

**Comment by jawaad-ahmad** [comment]
Should we have a **cross-reference** label in both repos in order to mark the issues as a cross-reference from another repo?

Should we have another label to indicate some sort of cross-repository change, so expect a corresponding issue in another repo to work another portion of the change? In the old repo we had multiple **Component** labels on the same issue to track this.

**Comment by ellerychan** [comment]
I don't think we need a **cross-reference** tag.  The end of the old thread will point the start of the new thread, and also the reverse.  The tag would only help if we wanted to find all cases of that, but why would we?  If the issue affects multiple repos then it would be good to link it somehow, so a tag inidicating the other repo(s) is probably appropriate.

**Comment by jawaad-ahmad** [comment]
How about **partial**?

Also what about to track any dependencies? For example if Issue 14 must be pulled before Issue 8.

**Comment by ellerychan** [comment]
Will putting the reference in the body as a link (e.g., #3) suffice for that?  Is there any GitHub mechanism that maps those relationships?  I don't want there to be so many tags.  It increases the chance that they will be applied inconsistently, which would decrease their usefulness.

**Comment by jawaad-ahmad** [comment]
None that I'm aware of; we can ignore for now and address if the need comes up.

**Comment by ellerychan** [comment]
Did the trial run.  See Issue #4.  It continues from brata-hsdc/brata#74.  I like it.  Straightforward and tidy.  If you agree, you could _close_ this issue.

**Comment by ellerychan** [comment]
Are we capturing these decisions somewhere for everyone else?  I've started a .md for it, but where should it go?  Wiki?  Tell me where, and I will place it there.  Or if you have already started something, yours is probably better, and I will just update it.

**Comment by jawaad-ahmad** [comment]
I think a **Process** page on the wiki would be appropriate. At least for the Master Server group. We can promote it to the other teams and get their buy-in before making it a global process.

**Comment by jawaad-ahmad** [comment]
References look great. How do we make them?

**Comment by ellerychan** [comment]
If you mean the links to the issues in the repositories, you just type the text, and GitHub recognizes it and makes the link.  Just '#' followed by the issue number for the current repo, and put the repo path on the front of it for other repos.

**Comment by ellerychan** [comment]
Added the Wiki to brata-hsdc/brata.masterserver.  You should see a new Wiki tab on the right side of your workspace.  I made it just editable by users with push access to the repo.

**Comment by jawaad-ahmad** [comment]
It appears GitHub issues only have two states: Open and Closed. We'll need to manage additional states via labels.

**Comment by jawaad-ahmad** [comment]
I added a bunch of states representing our current schema as a starting point; i don't think we need all these and i think we can rename a couple to make more sense for our workflow here.

**Comment by jawaad-ahmad** [comment]
I noticed @ellerychan opened brata-hsdc/brata#91 for part of this work.

**Comment by jawaad-ahmad** [comment]
I've been going through the existing issues trying to clean up the ones from last year. I know we said we'd pull-in the MS issues as we work them, but there are two problems with that.
1. At a glance, I don't know what work we have on the MS by looking at the MS issue tracker.
2. The MS-specific issues are cluttering the Brata issue tracker. Same for the others.

I went through and pulled out all the MS-specific issues and opened them on the MS issue tracker, leaving the general ones still on the Brata tracker, so there are still some tagged to `Component-MS` for now.

We still have the station issues to pull out. I'm not in a rush to do that at the moment because it'll mean keeping a third issue tracker's labels in-sync with our own. We've made good progress so far and I think it would be good to let the dust settle before beginning with a third tracker.

**Comment by ellerychan** [comment]
I agree with all that.

**Comment by jawaad-ahmad** [comment]
I moved a bunch over to the station repository with our current labels.

I created a `brata-framework` label in the brata issue tracker for BRATA-specific issues. There were only three.

The rest are all general issues that we need to go through some time soon.

There were a couple that were station-specific, but it looks like the work was already done and we need confirmation from @jaron42 that he did them and they're in the baseline. I didn't see any point migrating these over unless @jaron42 states there's still work to do. I tagged these with `question`.
